plugin/geoip/city.go

```diff
@@ -0,0 +1,60 @@
+package geoip
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/coredns/coredns/plugin/metadata"
+	"github.com/oschwald/geoip2-golang"
+)
+
+const defaultLang = "en"
+
+func (g GeoIP) setCityMetadata(ctx context.Context, data *geoip2.City) {
+	// Set labels for city, country and continent names.
+	if name, ok := data.City.Names[defaultLang]; ok {
```

Comment:
should we set an error in the label, when the language is not available for this city? 'no city found for 'en'' or something?

Reply:
My two cents,
To the best of my knowledge `en` language is always set in all name fields (city, country, continent) in the official `mmdb` databases, if it's not being set it would be a "bug" during database creation, of course I think this scenario needs to be handle.

The advantage I feel provides not setting the label (as it's done right now), is that is straightforward for consumers to know when the value of the label is not available and therefore not do anything with it, they simply load the ` metadata.ValueFunc` and if it's nil then it does not exist, the downside is that you don't know what's the reason for it to not exist.

In the contrary when we set the label to an error, as you suggested is not that straightforward to know for a consumer if the value is the real value name, or just an error. May cause consumers to rely and hard code the error text in their code, etc. The positive is that the error description is provided instead of simply not setting the label... although I'm not completely sure how a user would leverage it.

WDYT?